YellowNotes.Core/Utility/TokenUtility.cs

```diff
@@ -5,9 +5,26 @@
 
 namespace YellowNotes.Core.Utility
 {
-    public static class TokenParser
+    public static class TokenUtility
```

Comment:
do not make class static, that is totally bad habit!
drawback of static class it that we can't unit test them